mod-source-record-manager-server/src/test/java/org/folio/rest/impl/AbstractRestTest.java

```diff
@@ -172,6 +172,12 @@ public abstract class AbstractRestTest {
     .withAction(ActionProfile.Action.CREATE)
     .withFolioRecord(ActionProfile.FolioRecord.HOLDINGS);
 
+/*  private final MappingProfile marcHoldingsMappingProfile = new MappingProfile()
```

Comment:
is this intentional? the comment?

Reply:
@okolawole-ebsco  Good point. I will remove it.